phpBB/phpbb/cache/driver/file.php

```diff
@@ -584,4 +587,15 @@ function _write($filename, $data = null, $expires = 0, $query = '')
 
 		return $return_value;
 	}
+
+	/**
+	* Replace slashes in the file name
+	*
+	* @param string $name filename
+	* @return string the corrected name
+	*/
+	function replace_slashes($name)
```

Comment:
Following the previous discussion in https://github.com/phpbb/phpbb/pull/3453#discussion_r25686975, this function should probably called "clean_varname($name)".